packages/app-bridge/src/registries/commands/Connection.ts

```diff
@@ -0,0 +1,3 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const connection = (): { name: 'openConnection' } => ({ name: 'openConnection' });
```

Comment:
```suggestion
export const openConnection = (): { name: 'openConnection' } => ({ name: 'openConnection' });
```
Filename can be `Connection.ts` but the method should stay descriptive of the dispatch

Reply:
ah no i acutally didnt wanted to chagne the method, but idea magic just changed it xD